lib/node_modules/@stdlib/lapack/base/dlacn2/README.md

```diff
@@ -0,0 +1,275 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dlacn2
+
+> Estimate the 1-norm of a square, real matrix `A`, reverse communication is used for evaluating matrix-vector products.
```

Comment:
This description is hard to read, as "reverse communication..." is a separate clause. This should be improved.